src/Okta/Provider.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace SocialiteProviders\Okta;
+
+use SocialiteProviders\Manager\OAuth2\User;
+use Laravel\Socialite\Two\ProviderInterface;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+
+class Provider extends AbstractProvider implements ProviderInterface
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'OKTA';
+
+    /**
+     * Scopes defintions.
+     *
+     * @see http://developer.okta.com/docs/api/resources/oidc.html#scopes
+     */
+    const SCOPE_OPENID = 'openid';
+    const SCOPE_PROFILE = 'profile';
+    const SCOPE_EMAIL = 'email';
+    const SCOPE_ADDRESS = 'address';
+    const SCOPE_PHONE = 'phone';
+    const SCOPE_OFFLINE_ACCESS = 'offline_access';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [
+        'openid',
+        'profile',
+        'email',
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopeSeparator = ' ';
+
+    protected function getOktaUrl()
+    {
+        return config('services.okta.url');
```

Comment:
Use https://github.com/SocialiteProviders/Manager/blob/master/src/ConfigTrait.php#L46-L52 for additional configuration keys.